ibmcloud/image/verify.sh

```diff
@@ -54,6 +54,13 @@ while ! ibmcloud is instance $id --output json | jq -e '.status == "running"' >
     sleep 5
 done
 
+skip_verify_console=$SKIP_VERIFY_CONSOLE
+
```

Comment:
This line is no longer necessary.

Reply:
Ah because we won't have the unbound variable issue now of course - I've taken out this line too, thanks!